aten/src/ATen/native/cuda/ActivationSiluKernel.cu

```diff
@@ -30,13 +30,13 @@ void silu_kernel(TensorIteratorBase& iter) {
         gpu_kernel(iter, [] GPU_LAMBDA(scalar_t x) -> scalar_t {
           using opmath_t = at::opmath_type<scalar_t>;
           const opmath_t x_acc = static_cast<opmath_t>(x);
-          return x_acc / (opmath_t(1) + c10::cuda::compat::exp(-x_acc));
+          return x_acc / (opmath_t(1) + c10::complex<double>::cexp(-x_acc));
```

Comment:
```suggestion
          return x_acc / (opmath_t(1) + c10::cuda::compat::exp(-x_acc));
```

Reply:
Oh, sorry. I misunderstood. I changed the code

https://github.com/pytorch/pytorch/pull/89649/commits/44720c41641741d8298d2a15e5f0c80fe06b7235